schema/README.md

```diff
@@ -1,16 +1,11 @@
 # Schema Generator
 
 This folder contains a script for generating enigma-go based on a specific version of Qlik Associative Engine.
-The version to be used must be one of the published versions of the Qlik Associative Engine, see [here](https://hub.docker.com/r/qlikcore/engine/tags/).
+The Engine OpenRPC specification can be found in the schema folder `./schema/engine-rpc.json`.
```

Comment:
Better to link directly to the file since it is checked into the repo.

Reply:
👍